docs/advanced_usage.ipynb

```diff
@@ -305,7 +305,7 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "100%|███████████████████████████████████████████████████████████████████████████████████████████████████████| 28/28 [00:00<00:00, 34.33it/s]\n"
+      "100%|███████████████████████████████████████████████████████████████████████████████████████████████████████| 28/28 [00:00<00:00, 69.65it/s]\n"
```

Comment:
Do I notice correctly this actually is faster (more iterations per second).

Reply:
Jup, I also would assume so, although this is the zonal stats part, whereas my experiment with the profiler did not notice a real difference on the main part which is the model runs.